src/vinyldns/batch_change.py

```diff
@@ -161,13 +197,24 @@ def from_dict(d):
             created_timestamp=map_option(d.get('createdTimestamp'), parse_datetime),
             changes=[BatchChange.change_type_converters[elem['changeType']](elem) for elem in d.get('changes', [])],
             id=d['id'],
-            owner_group_id=d.get('ownerGroupId')
+            status=d['status'],
+            owner_group_id=d.get('ownerGroupId'),
+            owner_group_name=d.get('ownerGroupName'),
+            # approval_status=d['approvalStatus'],
```

Comment:
approval status is optional, so it would be `d.get['approvalStatus']`

Reply:
it's not optional, just realized I didn't add some of the fields in the JSON serializer in the API. Will make a PR for those.